lib/geordi/docker.rb

```diff
@@ -5,6 +5,9 @@
 module Geordi
   class Docker
     DOCKER_COMPOSE_FILE = 'docker-compose.yml'.freeze
+    VNC_DISPLAY = ':17'.freeze
```

Comment:
When L35 always invokes `launch_vnc_viewer` with an argument, how is this constant still needed? Should be removed IMHO.